content/SmallFixes/chunk29/kronstadt_chair_fix/kronstadt_chair_you_have_researched_sitting_fix.entity.patch.json

```diff
@@ -0,0 +1,62 @@
+{
+	"tempHash": "00A9B93A825F5BDC",
+	"tbluHash": "005BA5CA6AC00627",
+	"patch": [
+		{
+			"SubEntityOperation": [
+				"0e4a8142f0ad2c47",
+				{
+					"SetPropertyValue": {
+						"property_name": "m_mTransform",
```

Comment:
Are these two Transforms necessary for enabling the interaction?

Reply:
No. They're both unrelated items that GlacierKit marks as changed just by clicking on them.

Reply:
Yep that sounds about right.

Reply:
Ah yeah, that's happened to me before.